cors/src/main/java/io/helidon/cors/CorsSupportHelper.java

```diff
@@ -369,14 +370,68 @@ public Aggregator aggregator() {
         return aggregator;
     }
 
-    private boolean isRequestTypeNormal(CorsRequestAdapter<Q> requestAdapter, boolean silent) {
+    // For testing
+    static RequestTypeInfo isRequestTypeNormal(String originHeader, UriInfo requestedHostUri, boolean isRequestSecure) {
+        return RequestTypeInfo.create(originHeader, requestedHostUri, isRequestSecure);
+    }
+
+    /**
+     * Captures intermediate data and the final result in deciding whether a request is a normal (non-CORS) request or not.
+     * <p>
+     *     We want to use the intermediate results for clearer logging if that's turned on without having to recompute it.
+     * </p>
+     * @param originLocation full origin (including port)
```

Comment:
(Layman question: but not scheme?)

Reply:
The scheme is part of the origin location as well. Fixed comment.